cpp-ch/local-engine/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ include_directories(
         ${CMAKE_CURRENT_BINARY_DIR}/proto
         ${THRIFT_INCLUDE_DIR}
         ${CMAKE_BINARY_DIR}/contrib/thrift-cmake
+        ${CMAKE_BINARY_DIR}/contrib/llvm-project/llvm/include
```

Comment:
why this?

Reply:
if not add this, while compile code, will has the excpeiton 'llvm/IR/IRBuilder.h' file can not be found, as this is defined in `FunctionBinaryArithmetic.h`.